### PR TITLE
1180 - Fixed issue regarding incorrect splitter button tooltip

### LINKED ISSUE #1180: Splitter collapse button not localized, text not changing
**Describe the bug**
on the soho splitter component, the title text does not change on close, it also does not localize the text.

**To Reproduce**
1. Go to https://github.com/infor-design/enterprise/blob/6a02596a30ddcadf5cb59343bed5bc62a44bc160/src/components/splitter/splitter.js#L98

**Expected behavior**
the text should either be removed, or when closed change to "expand" it also needs to be localized
it would also be nice if it was a soho-tooltip

### PR DESCRIPTION
**Pull request by tmcastro**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**
To fix #1180 just removed the tooltip, the button already shows the direction in the icon and the tooltip is annoying because it popups up everytime

**Related github/jira issue (required)**:
Closes #1180

**Steps necessary to review your pull request (required)**:
- http://localhost:4000/components/splitter/test-collapse-button.html
- now no tooltip on the expand button

### CODE REVIEW

File: src/components/splitter/splitter.js [hunk 1]
```diff
@@ -105,19 +105,27 @@ Splitter.prototype = {
             if (splitter[0].offsetLeft <= 10) {
               self.splitTo(defaultOffset, parentHeight);
               $(this).addClass('rotate');
+
+              $splitterButton.tooltip({ content: 'Collapse' });
```
Comment (tmcconechy) [inline src/components/splitter/splitter.js]: Looks good but we can translate this so make these all 
```
Locale.translate('Collapse');
// and
Locale.translate('Expand');
```

File: src/components/splitter/splitter.js [hunk 2]
```diff
@@ -95,7 +95,7 @@ Splitter.prototype = {
 
       if (s.collapseButton) {
         let savedOffset = 0;
-        const $splitterButton = $('<button type="button" class="splitter-btn" id="splitter-collapse-btn" title="Collapse"><svg class="icon" focusable="false" aria-hidden="true" role="presentation"><use xlink:href="#icon-double-chevron"></use></svg></button>');
+        const $splitterButton = $('<button type="button" class="splitter-btn" id="splitter-collapse-btn"><svg class="icon" focusable="false" aria-hidden="true" role="presentation"><use xlink:href="#icon-double-chevron"></use></svg></button>');
```
Comment (davidcarlsonberg) [inline src/components/splitter/splitter.js]:
```suggestion
        const splitterButton = $('<button type="button" class="splitter-btn" id="splitter-collapse-btn"><svg class="icon" focusable="false" aria-hidden="true" role="presentation"><use xlink:href="#icon-double-chevron"></use></svg></button>');
```

File: src/components/splitter/splitter.js [hunk 3]
```diff
@@ -95,7 +95,7 @@ Splitter.prototype = {
...
         $splitterButton.appendTo(splitter);
```
Comment (davidcarlsonberg) [inline src/components/splitter/splitter.js]:
```suggestion
        splitterButton.appendTo(splitter);
```

File: src/components/splitter/splitter.js (hunk 1 above)
Comment (davidcarlsonberg) [inline src/components/splitter/splitter.js]:
```suggestion
              splitterButton.tooltip({ content: 'Collapse' });
```

File: src/components/splitter/splitter.js [hunk 4]
```diff
@@ -105,19 +105,27 @@ Splitter.prototype = {
...
               self.splitTo(defaultOffset, parentHeight);
               $(this).addClass('rotate');
+
+              $splitterButton.tooltip({ content: 'Collapse' });
             } else {
               savedOffset = splitter[0].offsetLeft;
               self.splitTo(0, parentHeight);
               $(this).removeClass('rotate');
+
+              $splitterButton.tooltip({ content: 'Expand' });
```
Comment (davidcarlsonberg) [inline src/components/splitter/splitter.js]:
```suggestion
              splitterButton.tooltip({ content: 'Expand' });
```

File: src/components/splitter/splitter.js [hunk 5]
```diff
@@ -105,19 +105,27 @@ Splitter.prototype = {
...
             }
           } else if (splitter[0].offsetLeft > 10) {
             savedOffset = splitter[0].offsetLeft;
             self.splitTo(0, parentHeight);
             $(this).removeClass('rotate');
+
+            $splitterButton.tooltip({ content: 'Expand' });
```
Comment (davidcarlsonberg) [inline src/components/splitter/splitter.js]:
```suggestion
            splitterButton.tooltip({ content: 'Expand' });
```

File: src/components/splitter/splitter.js [hunk 6]
```diff
@@ -105,19 +105,27 @@ Splitter.prototype = {
...
           } else {
             self.splitTo(savedOffset, parentHeight);
             $(this).addClass('rotate');
             savedOffset = 0;
+
+            $splitterButton.tooltip({ content: 'Collapse' });
```
Comment (davidcarlsonberg) [inline src/components/splitter/splitter.js]:
```suggestion
            splitterButton.tooltip({ content: 'Collapse' });
```

**Comment by tmcastro** [comment]
I've added your comment on the last update, guys.. but the test keeps failing. I can't replicate the build error on my machine.

**Pull request merged**